### DISCUSSION TITLE
Uh.. am I a moron? C210 with Firmware 230823 - Won't let me log into it!! Using correct password/username as instructed on the screen which asks for it.

### DISCUSSION THREAD

**Discussion by phuzzyday** [JurajNyiri/HomeAssistant-Tapo-Control#654] [Q&A]
Ok. The Camera password I made in Camera Settings/Advanced Settings/Camera Account is the one I am using.
I am getting 'Invalid Authentication Data'. I have tested the rstp stream with VLC as described, and it said it was 'unable to connect'. At one point, It did pause and ask me to enter the password again at one point before it did fail to connect again.
Let me re-check the firmware. It's running This one I flashed. 1.3.7_Build_230823. Shows in the app also. The camera is a V2 version. Now I haven't messed with opening ports, but this is just a home network, I do not believe the network firewall would block ports between devices on the same network...  Would it? I have combed the firewall/router settings, and haven't seen that choice.

**Comment by JurajNyiri** [comment]
Hi, first make sure you have followed all steps and understand them under FAQ point "I see error `Invalid authentication data. Make sure you have created your 3rd party account via Tapo app. You can also test if these credentials work via rtsp stream, for example VLC using link rtsp://username:password@IP Address:554/stream1` when I enter correct password"

**Reply by phuzzyday** [reply]
> Hi, first make sure you have followed all steps and understand them under FAQ point "I see error `Invalid authentication data. Make sure you have created your 3rd party account via Tapo app. You can also test if these credentials work via rtsp stream, for example VLC using link rtsp://username:password@IP Address:554/stream1` when I enter correct password"

Thanks - I see problems in areas excluding the Tapo Control software, so I can dig into other areas. Thanks for the advise.

**Comment by sanebg** [comment]
This Addon is totally useless. It's called "camera control," but to control the cameras and integrate it into Hassio, you have to go through 100 hoops. Better close the repository and move on. 8 months you cannot fix an essential issue. I already moved to Reolink cameras and I'll never look back to Tapo.

**Reply by JurajNyiri** [reply]
1. Factually wrong. If you read the information available you would know I have fix available since April. 

2. It is not an addon but an integration. 

3. Not 8 months. 

2. This is free and open source project. Go and fix it yourself and create new project extracting the token. 
3. This issue is for rtsp stream difficulties which have nothing to do with firmware issue for control. I know reading is hard. 
4. Banned. No other words needed.
